template/template.go

```diff
@@ -102,8 +102,21 @@ var response = parse.ActionNode{
 	},
 }
 
+var funcs = template.FuncMap{
+	"aggregate": func(v []*api.Message, role string) string {
```

Comment:
this works, wondering if there's a more Go-like name for it like `join`, and, against my usual suggestion, if we should make it more specific in case we'll regret it later (e.g. `joinMessages`)